### PR TITLE
New package: cdk-5.0-20190303

### PR DESCRIPTION
**Pull request by ndowens**
Closes: #17813

Signed-off-by: Nathan Owens <ndowens04@gmail.com>

### CODE REVIEW

File: srcpkgs/cdk/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# Upstream's ftp causes travis to timeout, possilby buildbot
+distfiles="https://fossies.org/linux/misc/cdk-${version}-${_date}.tgz"
+checksum=adad7870988d44b95df57722ab8dffc587d035183eb6b12a9500ebed4d8dba25
+
+post_install() {
+	vlicense COPYING
+}
+
+cdk-devel_package() {
+	makedepends="ncurses-devel"
```
Comment (xtraeme) [inline srcpkgs/cdk/template]: this is wrong... subpkgs don't have `makedepends` :-)

File: srcpkgs/cdk/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# Upstream's ftp causes travis to timeout, possilby buildbot
+distfiles="https://fossies.org/linux/misc/cdk-${version}-${_date}.tgz"
+checksum=adad7870988d44b95df57722ab8dffc587d035183eb6b12a9500ebed4d8dba25
+
+post_install() {
+	vlicense COPYING
+}
+
+cdk-devel_package() {
+	depends="ncurses-devel"
```
Comment (xtraeme) [inline srcpkgs/cdk/template]: `depends="${makedepends} ${sourcepkg}-${version}_${revision}"`

File: srcpkgs/cdk/template [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+_date=20190303
+wrksrc="cdk-${version}-${_date}"
+build_style=gnu-configure
+configure_args="--with-shared --with-ncurses=${XBPS_CROSS_BASE}/usr/lib"
+makedepends="ncurses-devel"
+short_desc="Curses Development Kit - widget library, enhanced version"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="GPL-2.0-only, BSD-4-Clause"
+homepage="https://invisible-island.net/cdk/"
+# Upstream's ftp causes travis to timeout, possilby buildbot
```
Comment (xtraeme) [inline srcpkgs/cdk/template]: who cares for ftp in 2020? get rid of this comment

File: srcpkgs/cdk/template [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+post_install() {
+	vlicense COPYING
+}
+
+cdk-devel_package() {
+	depends="ncurses-devel"
+	short_desc+=" -development files"
+	pkg_install() {
+		vmove usr/bin/cdk5-config
+		vmove usr/include
```
Comment (xtraeme) [inline srcpkgs/cdk/template]: no static archive?

Comment (ndowens) [reply srcpkgs/cdk/template]: It did not generate one with shared enabled

Comment (xtraeme) [reply srcpkgs/cdk/template]: get rid of it, we do care for static stuff... musl is our friend

Comment (ndowens) [reply srcpkgs/cdk/template]: I will see if I can make it gen a static as well later

File: srcpkgs/cdk/template [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+homepage="https://invisible-island.net/cdk/"
+distfiles="https://fossies.org/linux/misc/cdk-${version}-${_date}.tgz"
+checksum=adad7870988d44b95df57722ab8dffc587d035183eb6b12a9500ebed4d8dba25
+
+post_install() {
+	vlicense COPYING
+}
+
+cdk-devel_package() {
+	depends=-${sourcepkg}-${version}_${revision} ncurses-devel"
```
Comment (xtraeme) [inline srcpkgs/cdk/template]: missing `"` here buddy

Comment (xtraeme) [reply srcpkgs/cdk/template]: it's a variable remember... needs quoting due to blanks

File: srcpkgs/cdk/template [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+	vlicense COPYING
+}
+
+cdk-devel_package() {
+	depends="${sourcepkg}-${version}_${revision} ncurses-devel"
+	short_desc+=" -development files"
+	pkg_install() {
+		vmove usr/bin/cdk5-config
+		vmove usr/include
+		vmove "usr/lib/*.a"
```
Comment (xtraeme) [inline srcpkgs/cdk/template]: no shared library this time? sounds weird

Comment (ndowens) [reply srcpkgs/cdk/template]: Yep if we do static, we don't get shared. If I enable shared, no static. Hmm, may have idea, will have to dive into the build script.

Comment (xtraeme) [reply srcpkgs/cdk/template]: no way buddy. two of them or nothing

**Comment by ndowens** [comment]
Working on making both, even if I have to make the static manually

**Comment by xtraeme** [comment]
let upstream know wtf is going on

File: srcpkgs/cdk/template (hunk 6 above)
Comment (ndowens) [reply srcpkgs/cdk/template]: I have manually created the static library and installed.

**Comment by ndowens** [comment]
> let upstream know wtf is going on

Sent email asking if they could make it where both can be built.

**Comment by LinArcX** [comment]
Any update?

**Pull request closed without merging**